### PR TITLE
new design

### PR DESCRIPTION
**Pull request by rishub**
discussed with mike and henna about re-design
made the front page have 3 columns of posts and 12 posts per page (changed from 10 cause divisible by 3)
definitely could use some design feedback
also messed with some default configurations so lmk if that's ok

screen shot 2018-05-24 at 3 15 47 pm

### CODE REVIEW

**Comment by nathunsmitty** [comment]
I'm a fan!! Is this mobile tested, though?

**Comment by rishub** [comment]
yea it should be fine on mobile. it collapses to 1 column but i’ll send a screenshot when i’m on my computer

**Comment by rishub** [comment]
rip u right. didn’t commit it

**Comment by hennadialani** [comment]
screen shot 2018-06-03 at 12 11 22 pm 

looks fine except this being slightly off, mobile view looks good too!

**Comment by rishub** [comment]
screen shot 2018-06-03 at 12 36 10 pm
fixed

**Comment by rishub** [comment]
@zeehang can you review again

File: _sass/_layout.scss [hunk 1]
```diff
@@ -219,13 +219,23 @@
 .post-list {
   margin-left: 0;
   list-style: none;
+  display: flex;
```
Comment (nathunsmitty) [inline _sass/_layout.scss]: oh huh I probably would have done it in grid but cool to see that you can do a grid layout in flexbox also!

**Pull request merged**